app/routes/_index.tsx

```diff
@@ -32,6 +32,12 @@ export default function Index() {
         </Link>
       </Button>
 
+      <Button variant={"outline"}>
```

Comment:
```suggestion
      <Button variant={"outline"} asChild>
```
Add it to the other links as well. (I have a PR for this that @VGcod3 never merged)